src/main/java/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
The field 'code' in the enum 'Operation' should be final since it is not intended to be changed after the enum constants are created.